metrics/run/collect_metrics.go

```diff
@@ -211,7 +216,14 @@ func main() {
 		*inputGcsBucket)
 
 	readStartTime := time.Now()
-	runsWithLabels := base.FetchLatestRuns(*wptdHost)
+	var labelSet mapset.Set
+	if *labels != "" {
+		labelSet = mapset.NewSet()
+		for _, label := range strings.Split(*labels, ",") {
+			labelSet.Add(label)
+		}
+	}
+	runsWithLabels := base.FetchRuns(*wptdHost, "latest", labelSet)
```

Comment:
Nit: `FetchRuns` is merely a wrapper of the HTTP API, and it in fact turns the labels back into a string param again. The only current user of `FetchRuns` is `util/populate_dev_data.go` (and this PR will be the second one). I think it's probably cleaner to just pass a comma-separated string to `FetchRuns`, but I don't have strong preference.

Reply:
Will clean this up at a later date.. the creation of the set is a code-smell but sets are a good thing.